python/pyarrow/tests/strategies.py

```diff
@@ -105,9 +105,7 @@
     pa.duration,
     st.sampled_from(['s', 'ms', 'us', 'ns'])
 )
-interval_types = st.sampled_from(
-    pa.month_day_nano_interval()
-)
+interval_types = st.just(pa.month_day_nano_interval())
```

Comment:
Not familiar with Hypothesis. Looks okay to me.
@jorisvandenbossche , will you have a look at this change?